src/components/TopBar/TopBar.vue

```diff
@@ -46,6 +46,22 @@
 			</div>
 		</a>
 
+		<a v-if="showUpcomingEvent"
+			class="upcoming-event__card"
```

Comment:
nitpick: to follow BEM, because there is no `upcoming-event` block to have element `upcoming-event__card` in.

```suggestion
			class="upcoming-event"
```